src/functions/utils.js

```diff
@@ -9,4 +11,44 @@ const getPinnedAndUnpinnedNotes = (notes) => {
   return { pinnedNotes, unPinnedNotes };
 };
 
-export { getDate, getTime, getPinnedAndUnpinnedNotes };
+const validateEmail = (email) => {
+  const regularExpression =
+    /^([a-z\d-]+)@([a-z\d-]+)\.([a-z]{2,8})(\.[a-z]{2,8})?$/;
+  if (regularExpression.test(email)) {
+    return true;
+  } else {
+    toast.warning("Enter a valid email address");
+    return false;
+  }
+};
+
+const validatePassword = (password) => {
+  const regularExpression =
+    /^(?=.*[0-9])(?=.*[!@#$%^&*])[a-zA-Z0-9!@#$%^&*]{8,16}$/;
+  if (regularExpression.test(password)) {
+    return true;
+  } else {
+    toast.warning(
+      "Password must include a number, a uppercase letter, a lowercase letter and a special character"
```

Comment:
Even after entering one uppercase, one lower case, number and a special character, unable to signup, so if minimum  character limit is there, please add it in info message
![image](https://user-images.githubusercontent.com/29355667/174433293-6bdc51d7-a622-4735-bed5-467db1dd5a4a.png)

Reply:
Thanks for pointing it out